app/forms/schools/form_validator.rb

```diff
@@ -21,13 +21,27 @@ def initialize(trainee, non_search_validation: false)
       @fields = lead_school_form_fields.merge(employing_school_form_fields)
     end
 
+    def missing_fields
+      school_forms.flat_map do |form|
```

Comment:
This will need to be modified to be contained in an `[]`, following the latest merge to master (support for multiple degrees).  
https://github.com/DFE-Digital/register-trainee-teachers/blob/9babc1cba055c157e460f90a1e69654a15d9fbbd/app/forms/funding/form_validator.rb#L29-L36

Reply:
updated 👍